### ISSUE TITLE
Match Punctuation problems

### ISSUE THREAD

**Issue by jdegand** [sindresorhus/awesome-lint#217]
I didn't have an error until now for this.  

I think an array of arrays is expected (destructuring for left and right pairs is used) versus strings.

```js
[matchPunctuation, [['"', '"'], ['『', '』'], ['（', '）'], ['《', '》'], ['「', '」'], ['【', '】']]]
```

**Comment by sindresorhus** [comment]
What is the problem?

https://github.com/laysent/remark-lint-plugins/tree/master/packages/remark-lint-match-punctuation

**Comment by jdegand** [comment]
@sindresorhus Going to have to reopen this issue as the fix didn't work.

**Comment by DaltonSW** [comment]
This was introduced sometime after v2.1.0. `npx awesome-lint@2.1.0` doesn't give any warnings. I'm still experiencing the same thing on current version

@sindresorhus Can you reopen the issue?

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/awesome-lint/releases/tag/v2.2.1
